### PR TITLE
fix: broken builds when using wrapped components

### PR DESCRIPTION
**Pull request by sebastian-naicker**
Yo, had an issue, where the builds were failing, due to missing 'name' property when wrapping child components with HOC's. It fixed the issue locally for me. Happy to update if you notice any issues.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -13,9 +13,10 @@ function isReactFragment(openingElement) {
 function applyAttribute({ openingElement, t, name, options }) {
   if (!openingElement || isReactFragment(openingElement)) return
 
-  const isAttributeAlreadySet = openingElement.node.attributes.find(
-    node => node.name.name === options.attribute
-  )
+  const isAttributeAlreadySet = openingElement.node.attributes.find(node => {
+    if (!node.name) return // fix build issues for wrapped components
```
Comment (davesnx) [inline src/index.js]: Happy to add the condition, could you remove the comment or make it more generic?

**Review by davesnx — CHANGES_REQUESTED** [review]
Looks great, could you remove the package-lock?

File: src/index.js (hunk 1 above)
Comment (sebastian-naicker) [reply src/index.js]: Sure thing, just added for context for you :)

Comment (sebastian-naicker) [reply src/index.js]: Done :)

**Comment by davesnx** [comment]
Published under 1.5.1

**Comment by sebastian-naicker** [comment]
Thanks mate.

**Comment by shepitchakpavlo** [comment]
> Published under 1.5.1

The version is not available. The latest one is 1.5.0. @davesnx Could you check, please?

**Comment by davesnx** [comment]
Published under 1.6.0. The CI to autopublish doesn't work :cry:

**Pull request merged**